facilitator/src/aggregation.rs

```diff
@@ -387,7 +398,7 @@ impl<'a> BatchAggregator<'a> {
                     // server.aggregate
                     .context(format!(
                         "trace id {} failed to validate packets",
-                        self.trace_id
+                        &self.trace_id
```

Comment:
I think maybe errors should not include the trace ID in the message, because then they won't show up when we search for log statements by the JSON field "trace id". I think the thing to do would be to have any log line that logs an error include a trace ID. So for instance, this error returned by `BatchAggregator` would eventually get logged by [`facilitator.rs::aggregate_worker()`](https://github.com/abetterinternet/prio-server/blob/d1b991ec8733105a8a1d89176dd7b84f74215c75/facilitator/src/bin/facilitator.rs#L1166):

```
            match result {
                Ok(_) => queue.acknowledge_task(task_handle)?,
                Err(err) => {
                    error!("error while processing task {}: {:?}", task_handle, err);
                    queue.nacknowledge_task(task_handle)?;
                }
            }
```
`trace_id` is in scope there, so the above would become:
```
            match result {
                Ok(_) => queue.acknowledge_task(task_handle)?,
                Err(err) => {
                    error!("error while processing task {}: {:?}", task_handle, err; "trace id" => trace_id);
                    queue.nacknowledge_task(task_handle)?;
                }
            }
```